tests/e2e-instrumentation/instrumentation-java-multicontainer/00-install-instrumentation.yaml

```diff
@@ -9,11 +9,9 @@ spec:
     - name: OTEL_EXPORTER_OTLP_ENDPOINT
       value: http://localhost:4317
     - name: OTEL_EXPORTER_OTLP_TIMEOUT
-      value: "20"
+      value: "20000"
```

Comment:
this is to account for the collector not being ready fast enough

Reply:
What happens if it's not ready? Shouldn't this work correctly either way?